extensions/amp/content/src/content/jcr_root/apps/core/wcm/extensions/amp/components/search/v2/search/clientlibs/amp/js/search.js

```diff
@@ -0,0 +1,62 @@
+/*******************************************************************************
+ * Copyright 2022 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+(function() {
+    "use strict";
+    window.onload = updateSearchResultsStatusMessageElement;
+
+    var selectors = {
+        searchComponent: ".cmp-search",
+        searchInput: ".cmp-search__input",
+        searchResultsContainer: ".i-amphtml-autocomplete-results",
+        itemInSearchResults: ".i-amphtml-autocomplete-item",
+        searchResultsStatusMessage: ".cmp_search__info"
+    };
+
+    function updateSearchResultsStatusMessageElement() {
```

Comment:
Isn't it the same logic as for the Search v2? Can't it be reused here?

Reply:
It's not the same logic, just the names of the methods.